sormas-ui/src/main/java/de/symeda/sormas/ui/immunization/ImmunizationDataView.java

```diff
@@ -72,9 +72,11 @@ protected void initView(String params) {
 		final boolean deleted = FacadeProvider.getImmunizationFacade().isDeleted(uuid);
 
 		if (deleted) {
-			layout.disable(CommitDiscardWrapperComponent.DELETE_UNDELETE);
+			editComponent.addButtonToExcludedList(CommitDiscardWrapperComponent.DELETE_UNDELETE);
```

Comment:
would be better to introduce a method in LayoutWithSidePanel that handles all this if/elseif-s so it would not be duplicated in all View classes